src/WORKSPACE

```diff
@@ -183,35 +183,74 @@ http_archive(
 
 git_repository(
     name = "rules_python",
-    commit = "a0fbf98d4e3a232144df4d0d80b577c7a693b570",
+    commit = "c8c79aae9aa1b61d199ad03d5fe06338febd0774",
     remote = "https://github.com/bazelbuild/rules_python.git",
     shallow_since = "1586444447 +0200",
 )
 
-load("@rules_python//python:repositories.bzl", "py_repositories")
+# Based on the hermetic Python 3 guide found here
+# https://thethoughtfulkoala.com/posts/2020/05/16/bazel-hermetic-python.html
 
-py_repositories()
+# Special logic for building python interpreter with OpenSSL from homebrew.
+# See https://devguide.python.org/setup/#macos-and-os-x
+_py_configure = """
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    ./configure --prefix=$(pwd)/bazel_install --with-openssl=$(brew --prefix openssl)
```

Comment:
we probably won't support `macOS`, do we need this check here?

Reply:
that was copypasted from the hermetic python guide, probably unnecessary